src/definitions/behaviors/form.js

```diff
@@ -1053,7 +1053,7 @@ $.fn.form = function(parameters) {
                 }
                 else if(isCheckbox) {
                   module.verbose('Setting checkbox value', value, $element);
-                  if(value === true) {
+                  if(value === true || value === 1) {
```

Comment:
I think limiting to boolean is reasonable to avoid confusing on truthy values.
For numbers, an usage-site type conversion like `$chekbox.checkbox("set value", value === 1)` is suffiient.

I am proposing this to avoid limitless widening acceptable values in case someone request "Please accept string 'true' too".

Reply:
Good point, but for checkbox in this only case, i wouldn't care so much, as it's optional and 0/1 used as false/true is widely used. 
I tested before using `Boolean(value) === true`, but then anything else also was true (even '0' is true then!), So i would definaltey not accept strings here, just because its not common.

But i am fine to decline the issue request and don't merge this if most of the other contributors also think we should stick with pure boolean values.

Reply:
I agree both but I think leaving it as just boolean and 1/0 should be fine and should reduce any issues we could face.

Reply:
I've ended up trying to write my own version roughly based on this example [https://6pac.github.io/SlickGrid/examples/example-composite-editor-item-details.html](url). It's more of a form controler.
(my original fix was a !!)